public_html/wp-content/mu-plugins/wcorg-json-api.php

```diff
@@ -361,9 +361,11 @@ function wcorg_json_v2_compat( $request ) {
 	}
 
 	// Route v2 requests to Core handler
-	$request->query_vars['rest_route'] = $request->query_vars['json_route'];
-	unset( $request->query_vars['json_route'] );
-	$request->matched_query = preg_replace( '#^json_route=(.+)$#', 'rest_route=$1', $request->matched_query );
+	if ( isset( $request->query_vars['json_route'] ) ) {
```

Comment:
`MUST` - Why is this change necessary? Because the `json-rest-api` plugin isn't available in the w.org repo anymore? Do the v1 and v2 endpoints still work on your sandbox with this change?

Reply:
Yup, thats because `json-rest-api` plugin isn't available in w.org repo. I tested out few `v2` endpoints, and some `json_route` endpoints like `?json_route=/posts/types/page` and they seem to work.

Is there anything specific you have in mind that we should test?

Reply:
Nope, that sounds good to me. I think this should be a separate commit from the other Docker stuff, and the commit message should document that it's because the plugin isn't in the repo (publicly) anymore. Looks good though :)